### PR TITLE
fix: add span tag to label

### PR DESCRIPTION
**Pull request by li-jia-nan**
fix: add span tag to label

### CODE REVIEW

File: src/TabNavList/TabNode.tsx [hunk 1]
```diff
@@ -53,6 +53,14 @@ function TabNode({
     });
   }
 
+  const lableNode = React.useMemo<React.ReactNode>(() => {
+    if (typeof label === 'string') {
+      return <span>{label}</span>;
```
[the remark this answers is not in the document]
Comment (li-jia-nan) [reply src/TabNavList/TabNode.tsx:58]: done

File: src/TabNavList/TabNode.tsx [hunk 2]
```diff
@@ -51,6 +51,12 @@ const TabNode: React.FC<TabNodeProps> = props => {
     editable.onEdit('remove', { key, event });
   }
 
+  // for compatibility https://developer.mozilla.org/zh-CN/docs/Web/CSS/:only-child
```
Comment (MadCcc) [inline src/TabNavList/TabNode.tsx:54]:
```suggestion
```
留测试不留 comment

Comment (li-jia-nan) [reply src/TabNavList/TabNode.tsx]: done

File: tests/index.test.tsx [hunk 3]
```diff
@@ -638,4 +638,14 @@ describe('Tabs.Basic', () => {
     await waitFakeTimer();
     expect(container.querySelector('.rc-tabs-ink-bar')).toHaveStyle({ width: '18px' });
   });
+
+  it('support only-child', () => {
```
Comment (MadCcc) [inline tests/index.test.tsx:642]:
```suggestion
  it('Add span to text label when have icon', () => {
```

**Review by MadCcc — APPROVED** [review]

**Pull request merged**